lib/jwt.js

```diff
@@ -186,8 +186,8 @@ function verify(input, key, method, type, signature) {
 function sign(input, key, method, type) {
   var base64str;
   if(type === "hmac") {
-    hmac = cjshmac[method](input, key);
-    base64str = crypto.enc.Base64.stringify(hmac)
+   const hmac = cjshmac[method](input, key);
+   var base64str = crypto.enc.Base64.stringify(hmac)
```

Comment:
Variável declarada duas vezes, basta substituir `var base64str` por `base64str`.

Reply:
Hi ! you r right, it's fixed with the last commit.